core/upgrade/upgrades.php

```diff
@@ -928,6 +931,22 @@ public static function _v_3_2_0_migrate_breakpoints_to_new_system( $updater, $in
 		return self::move_settings_to_kit( $callback, $updater, $include_revisions );
 	}
 
+	public static function _v_3_5_0_remove_old_elementor_scheme() {
+		global $wpdb;
+
+		$key = Base::SCHEME_OPTION_PREFIX;
+
+		$wpdb->query( "DELETE FROM {$wpdb->options} WHERE option_name LIKE '{$key}%';" ); // phpcs:ignore
```

Comment:
@kobizz Please approve

Reply:
If there is an old site that has not been migrated from the scheme system to the global system yet, it won't be able to do it anymore.

Reply:
It is firstly upgraded to a kit, the upgrades are orders by the versions.